openhtf/plugs/user_input.py

```diff
@@ -73,20 +73,20 @@ def Stop(self):
     """Mark this ConsolePrompt as stopped."""
     self._stopped = True
     if not self._answered:
-      print "Nevermind; prompt was answered from elsewhere."
+      print("Nevermind; prompt was answered from elsewhere.")
 
   def run(self):
     """Main logic for this thread to execute."""
     try:
       if platform.system() == 'Windows':
         # Windows doesn't support file-like objects for select(), so fall back
         # to raw_input().
-        response = raw_input(self._message + '\n\r')
+        response = input(self._message + '\n\r')
```

Comment:
This 2to3 change means we need `from builtins import input` at the top for Python2.
http://python-future.org/compatible_idioms.html#raw-input